### PR TITLE
Update JB playlist link to new 4.3 series

### PR DESCRIPTION
**Pull request by benlumley**
Thought it'd be better to have this playlist link in now?

If you want the change; not sure on how to add to other languages? Do i just edit their json file ? Or do I have to do it via crowdin ? Happy to do it if someone can point me the right way.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: locales/en/messages.json
Comment (haslinghuis) [inline locales/en/messages.json:779]: Thanks! This link does not direct to the right video playlist (probably because there is only one part yet).
This is the link I'm using:
`https://www.youtube.com/watch?v=LkBWRiEGKTI&list=PLwoDb7WF6c8nT4jjsE4VENEmwu9x8zDiE`
Might fix itself when a second video is added to the list. So let's wait for JB follow up video(s).

**Comment by Asizon** [comment]
Also @benlumley  take acount that the links in configurator should be generic for all existing versions, Configurator not support only 4.3 version, adding a link for each version is not the way.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by benlumley** [comment]
> Also @benlumley take acount that the links in configurator should be generic for all existing versions, Configurator not support only 4.3 version, adding a link for each version is not the way.

aha. a very good point. perhaps wanna can this idea then.... ?

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
JB has added another part. Now the link works as intended.

**Comment by Asizon** [comment]
@haslinghuis the links in configurator should be generic for all existing versions, Configurator not support only 4.3 version, adding a link for each version is not the way.

**Comment by McGiverGim** [comment]
> Maybe we can add the link somewhere when fc is conected, based on detected firmware version.

I think this complicates the code, and gives doubts when no FC is connected, producing different experience to the user, than can lead to confusion.

**Comment by Asizon** [comment]
> > Maybe we can add the link somewhere when fc is conected, based on detected firmware version.
> 
> I think this complicates the code, and gives doubts when no FC is connected, producing different experience to the user, than can lead to confusion.

Yes, i mean to put it somewhere when conected to ui,in some tab, not in the main page @McGiverGim

**Review by limonspb — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by daleckystepan — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
